src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+
+@Entity
```

Comment:
What table name is generated here? Actor? If so, you should specify the name of each model table.

Reply:
make as suggested